Gui/opensim/view/src/org/opensim/view/experimentaldata/ExperimentalMarker.java

```diff
@@ -86,7 +87,9 @@ public void generateDecorations(boolean fixed, ModelDisplayHints hints, State st
     @Override
     void updateDecorations(ArrayDouble interpolatedStates) {
         int idx = getStartIndexInFileNotIncludingTime();
-        setPoint(new double[]{interpolatedStates.get(idx)/1000., interpolatedStates.get(idx+1)/1000., interpolatedStates.get(idx+2)/1000.});
+        setPoint(new double[]{interpolatedStates.get(idx)/conversion, 
```

Comment:
point data are held in `interpolatedStates`? That is confusing.

Reply:
@aseth1 the names come from using the Stroage class which has methods getStateVector etc. The meaning has shifted over the years so I understand the confusion but this should be taken care of when we switch to using tables in the future.

Reply:
`interpolatedStates` is an input argument for a method in this `ExperimentalMarker` class. Seems the author of this class has total control here to name the input argument whatever he likes. I suggest renaming it since it is misleading as to what `updateDecorations()` is supposed to be doing, which is to simply render marker data! Please rename it to `rowOfMarkerData` (or something like that) and add a comment about its layout, e.g. `idx` is a column index in the row of data - `getStartIndexInFileNotIncludingTime()` is not very informative, considering there is no *file* in the context of this class AFAICT.

Reply:
@aseth1 Actually there's a file that the trc data is read from as a Storage object. I agree we can rename the agument here but that would be more confusing since the method it overrides has the same signature. All this stems from using Storage that has StateVector as dataRow which has been in use since version 1.0.